src/client/_clientwindow.py

```diff
@@ -870,12 +874,26 @@ def saveWindow(self):
         util.settings.setValue("geometry", self.saveGeometry())
         util.settings.endGroup()
 
+    def show_autojoin_settings_dialog(self):
+        autojoin_channels_list = []
+        autojoin_channels_list = Settings.get('chat/auto_join_channels')
+        text_of_autojoin_settings_dialog = """
+        Enter the list of channels you want to autojoin at startup, separated by ;
+        For example: #poker;#newbie
+        To disable autojoining channels, leave the box empty and press OK.
+        """
+        channels_input_of_user, ok = QInputDialog.getText(self, 'Set autojoin channels',
+        text_of_autojoin_settings_dialog, QLineEdit.Normal, ';'.join(autojoin_channels_list))
```

Comment:
Give this line one more indentation level and we're good.

Reply:
ok done